src/app/components-small/template-details/template-details.component.ts

```diff
@@ -28,25 +29,20 @@ export class TemplateDetailsComponent implements OnInit {
     @Input() templateLink: ApiLink | null = null;
     @Input() tabLink: ApiLink | null = null;
 
-    locations: Location[] = [
-        { value: "workspace", description: "Workspace (appears in the plugin sidebar)" },
-        { value: "experiment-navigation", description: "Experiment Navigation (appears in the top navigation bar)" }
-    ];
-
     private initialValues = {
         name: "",
         description: "",
         sortKey: 0,
         filterString: "{}",
-        location: this.locations[0].value
+        location: TAB_GROUP_NAME_OVERRIDES["workspace"]
```

Comment:
Should this not be "workspace"? (this uses the human readable version of the location)